src/main/java/io/swagger/swaggerhub/plugin/requests/SaveSCMPluginConfigRequest.java

```diff
@@ -0,0 +1,248 @@
+package io.swagger.swaggerhub.plugin.requests;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+/**
+ * Configuration class used to store details required to create a request to SwaggerHub to create an SCM integration plugin.
+ * The inner builder class follows the builder pattern which aids building the SaveSCMPluginConfigRequest object.
+ * {@link SCMIntegrationPluginConfiguration} represents the object when being sent as part of a request body.
+ */
+public class SaveSCMPluginConfigRequest {
+
+    /*
+    Path Params
+     */
+    private String apiOwner;
+    private String api;
+    private String version;
+    private String oas;
+
+    /*
+    Request body params
+     */
+    private String scmProvider;
+    private String token;
+    private String repository;
+    private String repositoryOwner;
+    private String enabled;
+    private String branch;
+    private String syncMethod;
+    private String outputFile;
+    private String target;
+
+    public SaveSCMPluginConfigRequest(Builder builder) {
```

Comment:
I think typically in Builder pattern you make the Constructor of the Object private, so you force callers to use the Builder... not saying this is wrong, but if it has a builder it should use it.

Reply:
Good spot! My error 👍